### DISCUSSION TITLE
ISO & Currency

### DISCUSSION THREAD

**Discussion by SANKARAMDAS** [ParanoidUser/yolo#730] [Galaxy Trivia] [answered]
United Arab Emirates   -
Switzerland -
Georgia -
Faroe Islands -
India -
Romania -
Samoa -
Togo -

**Comment by peterbaccaro** [comment, marked as answer]
Here are the ISO codes and currencies for the countries you requested:

**United Arab Emirates**
- ISO code: AE
- Currency: UAE dirham (AED)

**Switzerland**
- ISO code: CH
- Currency: Swiss franc (CHF)

**Georgia**
- ISO code: GE
- Currency: Georgian lari (GEL)

**Faroe Islands**
- ISO code: FO
- Currency: Danish krone (DKK)

**India**
- ISO code: IN
- Currency: Indian rupee (INR)

**Romania**
- ISO code: RO
- Currency: Romanian leu (RON)

**Samoa**
- ISO code: WS
- Currency: Samoan tala (WST)

**Togo**
- ISO code: TG
- Currency: West African CFA franc (XOF)
